recipe/meta.yaml

```diff
@@ -1,8 +1,8 @@
-{% set version = "5.5.1068" %}
+{% set version = "5.5.112" %}
```

Comment:
How can we go back on the patch version here? Should the minor version have been bumped?

Reply:
Oops, fixed here: https://github.com/conda-forge/kaldi-feedstock/pull/45, sorry about that!